src/Aspire.Dashboard/Otlp/Model/OtlpApplication.cs

```diff
@@ -198,7 +198,7 @@ public static string GetResourceName(OtlpApplication app, List<OtlpApplication>
                 count++;
                 if (count >= 2)
                 {
-                    return app.InstanceId;
+                    return $"{item.ApplicationName} ({OtlpHelpers.TruncateString(app.InstanceId, maxLength: 7)})";
```

Comment:
nit : Can we add the length of 7 to constants since the ids are referenced in multiple files?